src/clone-node.ts

```diff
@@ -118,7 +127,7 @@ function cloneCSSStyle<T extends HTMLElement>(nativeNode: T, clonedNode: T) {
     targetStyle.cssText = sourceStyle.cssText
     targetStyle.transformOrigin = sourceStyle.transformOrigin
   } else {
-    toArray<string>(sourceStyle).forEach((name) => {
+    getStyleProperties(options).forEach((name) => {
```

Comment:
Instead of allocating a new array with all the property names for each node in order call `.forEach` on it, it would be even faster to just use a good-old `for` loop to iterate over all the properties. Since `CSSStyleDeclaration` is "array-like", it works well:

```ts
const propertyNames = options.includeStyleProperties ?? sourceStyle;
for (let i = 0; i < propertyNames; ++i) {
  const name = propertyNames[i];
  let value = sourceStyle.getPropertyValue(name);
  // ...
}
```